app/views/registrations/new.html.erb

```diff
@@ -27,92 +28,12 @@
   <div class="flex items-center gap-2">
     <%= f.check_box :terms_and_service, class: "w-4 h-4 text-teal-600 border-gray-300 rounded focus:ring-teal-500" %>
     <span class="text-sm text-gray-700">
-      I have read and understood the 
-      <a href="#" id="open-modal" class="text-teal-600 hover:underline">Terms and Services</a>
+      <%= t('terms_and_services.agreement_text') %>
```

Comment:
I18n yorumum burası için de geçerlidir

Reply:
Ayrıca burada span yerine label kullanalım ki tıklantığında checkbox'ı trigger etsin